dags/infra/sns_topics.py

```diff
@@ -0,0 +1,14 @@
+"""
+# Variables provided from infra to dags
+"""
+from airflow.models import Variable
+
+LANDSAT_SYNC_SNS_TOPIC_ARN = Variable.get(
+    "landsat_sns_topic",
+    "arn:aws:sqs:af-south-1:565417506782:deafrica-prod-af-eks-index-landsat-scene",
```

Comment:
how this bucket got provisioned?

Reply:
@NikitaGandhi this isn't a bucket, this is a topic, which the SQS indexing signs and receives messages from.